MVCapplication/src/main/webapp/WEB-INF/resources/js/listener.js

```diff
@@ -1,3 +1,9 @@
+var removeFunc = function () {
+    $("#notification").html(removeAll);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Move the declaration of "removeAll" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)